### PR TITLE
fix: added wrapper for slider [ci visual]

### PR DESCRIPTION
**Pull request by Smiranin**
## Related Issue
Closes SAP/fundamental-styles#2503

## Description
Added wrapper for slider to prevent getting out of the strip edges of slider track. (look related issue)
Also fixed position of "before element" of handle for rtl.

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: src/slider.scss [hunk 1]
```diff
@@ -148,3 +148,23 @@ $handle-hit-area-mobile-dimensions: 2.75rem;
     }
   }
 }
+
+.#{$block}-wrapper {
```
[the remark this answers is not in the document]
Comment (Smiranin) [reply src/slider.scss:152]: No, it is the parent element of the `.fd-slider`.

File: src/slider.scss [hunk 2]
```diff
@@ -148,3 +148,23 @@ $handle-hit-area-mobile-dimensions: 2.75rem;
...
+  padding-left: 0.625rem;
+  padding-right: 0.625rem;
```
[the remark this answers is not in the document]
Comment (Smiranin) [reply src/slider.scss]: Done

File: src/slider.scss [hunk 3]
```diff
@@ -148,3 +148,23 @@ $handle-hit-area-mobile-dimensions: 2.75rem;
...
+
+  &--lg {
+    padding-left: 0.8125rem;
+    padding-right: 0.8125rem;
```
[the remark this answers is not in the document]
Comment (Smiranin) [reply src/slider.scss]: +

File: src/slider.scss [hunk 4]
```diff
@@ -148,3 +148,23 @@ $handle-hit-area-mobile-dimensions: 2.75rem;
...
+  }
+}
+
+[dir='rtl'] {
```
[the remark this answers is not in the document]
Comment (Smiranin) [reply src/slider.scss]: +

File: src/slider.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (Smiranin) [reply src/slider.scss]: Updated changes. Remove all changes from html and removed wrapper. Added styles for fd-slider class

File: src/slider.scss [hunk 5]
```diff
@@ -16,9 +16,13 @@ $handle-hit-area-mobile-dimensions: 2.75rem;
 
   min-width: 4rem;
   max-width: 100%;
-  padding: 1rem 0;
+  padding: 1rem 0.625rem;
   position: relative;
 
+  &--lg-x-paddings {
```
Comment (InnaAtanasova) [inline src/slider.scss:22]: Please rename it to just `&--lg`. You can add in the documentation (the text above the example for mobile mode something like:
"To add horizontal paddings of 0.625rem use the <code>fd-slider--lg</code> modifier class".

**Review by InnaAtanasova — APPROVED** [review]

**Review by N1XUS — APPROVED** [review]

**Review by dimamarksman — APPROVED** [review]

**Pull request merged**